src/test/lib/sql/sql_translator_test.cpp

```diff
@@ -2556,8 +2556,7 @@ TEST_F(SQLTranslatorTest, Execute) {
 
 TEST_F(SQLTranslatorTest, ExecuteWithoutParams) {
   const auto prepared_lqp =
-  AggregateNode::make(expression_vector(), expression_vector(min_(int_float_a)),
-    stored_table_node_int_float);
+      AggregateNode::make(expression_vector(), expression_vector(min_(int_float_a)), stored_table_node_int_float);
```

Comment:
I think we should keep the original code here and disable clang-format for that code.